lms/templates/commerce/provider.underscore

```diff
@@ -0,0 +1,13 @@
+<p class="bold_param">
+  <%= interpolate(gettext("You still need to see visit %s to complete the credit process."), [interpolate("<a href='%s'>%s</a>", [provider_url, display_name]) ]) %>
+  <span><%= interpolate("<img src='%s' alt='%s'></image>", ["", display_name]) %></span>
+</p>
+<p>
+  <%= interpolate(gettext("In order to learn credit, %s requires learner to:"), [display_name]) %>
```

Comment:
I haven't seen this text before and I'm not sure what the context is here - ? @wedaly, I worked on some similar text recently; is this part of that batch of text?

In any case, I'd suggest at least changing this to "To earn course credit from %, follow these steps."